### PR TITLE
docs: applications Corrected wrong path in documentation

### PR DESCRIPTION
**Pull request by ProTofik**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Extensions/Applications.md [hunk 1]
```diff
@@ -258,7 +258,7 @@ options {
 3: Restart your bind9/named after changing the configuration.
 
 4: Verify that everything works by executing `rndc stats && cat
-/var/run/named/stats`. In case you get a `Permission Denied` error,
+/var/cache/bind/stats`. In case you get a `Permission Denied` error,
```
Comment (murrant) [inline doc/Extensions/Applications.md:261]: So, you changed a path that is correct for some OS/distros to a path that is correct for other OS/distros?

Comment (ProTofik) [reply doc/Extensions/Applications.md:261]: Step 1 says to create the file in /var/cache/bind/stats and set its permissions, so I just changed point 4 to match it.

Comment (murrant) [reply doc/Extensions/Applications.md:261]: Sounds good, reading snippets on PRs doesn't always give the full context.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-64-release-changelog-may-2020/12290/1

**Pull request merged**